packages/database/lib/modular/index.js

```diff
@@ -1,4 +1,5 @@
 import { getApp } from '@react-native-firebase/app';
+import { firebase } from '..'; // Needed for increment() method
```

Comment:
this brings more of the namespaced stuff into the new modular implementation...is it possible to import the ServerValue directly and use it directly (below)? There should be a way to import that ServerValue or we have bigger problems (which may also be the case: #8329 is currently a problem it appears - resolving both at once would be a big win

https://github.com/invertase/react-native-firebase/blob/f06528a64bbcf3489e6a1f3df997f3a6f0b5fd41/packages/database/lib/DatabaseStatics.js#L23

Reply:
@mikehardy
Adding

```js
import DatabaseStatics from '../DatabaseStatics';

const { ServerValue } = DatabaseStatics;
```
seems to work. Is this what you had in mind?

Reply:
@SelaseKay  Yep, that's what I had in mind! Small difference in practice but it is thinking forward to the time when the non-modular index.js / index.d.ts files go away. This has it already more set up for the future

Reply:
@mikehardy Nice. I've added the necessary changes.

Reply:
I believe this tweak could the reason this PR doesn't seem to have fixed the issue for us - at least, it's the best theory I've got.

More info: https://github.com/invertase/react-native-firebase/issues/8375#issuecomment-2738875112